en/releases/main.md

```diff
@@ -89,21 +89,20 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
 This release contains a major rework for the rover support in PX4:
 
 - Complete restructure of the [rover related documentation](../frames_rover/index.md).
-- New firmware build specifically for [rovers](../frames_rover/index.md#flashing-the-rover-build) ([PX4-Autopilot#22675](https://github.com/PX4/PX4-Autopilot/pull/22675)).
-- New module dedicated to [differential rovers](../frames_rover/differential.md) ([PX4-Autopilot#22402](https://github.com/PX4/PX4-Autopilot/pull/22402), [PX4-Autopilot#23430](https://github.com/PX4/PX4-Autopilot/pull/23430) and [PX4-Autopilot#23629](https://github.com/PX4/PX4-Autopilot/pull/23629))
+- New firmware build specifically for [rovers](../frames_rover/index.md#flashing-the-rover-build).
```

Comment:
This all looks good thank you. Is it deliberate to remove the PR links? Too many to track?

Reply:
Yes, i removed them because there were so many that it makes the realease notes harder to read.
My thinking was that since it will be a completely new "feature" coming with the next release we can just add it as one block. For everything after the initial release I would of course link the PRs again.

But if you think we need the PR links I can of course add them back in.

Reply:
I think it is fine to omit them, though when we next do a release I will doubtless ask "did we document this" because I won't remember or be able to look up the specific set of ids that we have covered. If we assume we have documented all needed docs work for rover then we're good to remove..